model/mobile_trigger.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+)
+
+const (
+	ScopeMobile = "mobile"
+	ScopeWebApp = "webapp"
+)
+
+type PluginIntegration struct {
```

Comment:
The struct name should be consistent with the route. Blocking here until https://github.com/mattermost/mattermost-server/pull/14906/files#r449028540 is resolved.

Reply:
Is `PluginIntegration` the name that was decided?